### PR TITLE
Update PR template to include 'Close' keyword

### LINKED ISSUE #922: Documentation:  Update PR checklist and other documentation around Issue Closing
### Please select a category the issue is focused on?

Developer Guides

### Let us know where something needs a refresh or put your idea here!

WE are not following the correct way to have the issue close automatically when merging into main.  Apparently keywords are needed to 

Updates are needed for:
- [x] PR Checklist
- [ ] Developer Guides [3 of them?]
- [ ] ?

https://stackoverflow.com/questions/12235620/automatically-closing-issue-from-pull-request-in-github

### PR DESCRIPTION
**Pull request by thomas-neitmann**
Close #922 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

**Review by bms63 — CHANGES_REQUESTED** [review]
Could we put something in the News.md checkbox.  Something around it being relevant to the user-base?  I saw some folks following it very strictly about things like fixing typos and links.

File: vignettes/git_usage.Rmd [hunk 1]
```diff
@@ -116,8 +116,8 @@ Once the review is completed, the reviewer will merge the Pull Request and the
 feature branch will automatically be deleted.
 
 After merging the Pull Request the corresponding issue must be moved to the
-"Sprint Done" column on the "admiral scrum board" by the developer. The issue
-must not be closed. It will be closed automatically once the `devel` branch is
+"Sprint Done" column on the "admiral scrum board" by the developer. **The issue
+must not be closed**. It will be closed automatically once the `devel` branch is
```
Comment (bms63) [inline vignettes/git_usage.Rmd:120]: Possible to add a Screen shot of where this is on the Issue?

image

**Comment by thomas-neitmann** [comment]
@bms63 I don't think this is NEWS-worthy. I would only put code changes and updates to user-facing documentation rather than dev guides.

**Comment by bms63** [comment]
> @bms63 I don't think this is NEWS-worthy. I would only put code changes and updates to user-facing documentation rather than dev guides.

I was saying people should not put things in that are not New-worthy.  Like this! :)   the template just says update News, but should say Update `News.md` if relevant to users of admiral?  Not sure how to phrase it.

File: vignettes/git_usage.Rmd (hunk 1 above)
Comment (bms63) [reply vignettes/git_usage.Rmd:120]: okay i will make get a screen shot.

Comment (thomas-neitmann) [reply vignettes/git_usage.Rmd:120]: Got it! Will try my best to phrase it right.

**Comment by bms63** [comment]
Can we hold off on doing this until the admiral board and cards have had their names updated? #991

**Comment by bms63** [comment]
Hey Thomas.  I uploaded some dark mode images and changed up the wording to match the current project board cards.

**Review by bms63 — APPROVED** [review]

**Pull request merged**